### PR TITLE
GA4GH updates for 86

### PR DESCRIPTION
**Pull request by sarahhunt**
Updates to handle the features compliance data in GFF.

Requires https://github.com/Ensembl/ensembl-rest_private/pull/9

### CODE REVIEW

**Comment by lairdm** [comment]
There's some GA tests failing, could you take a peek at those please?

**Comment by sarahhunt** [comment]
Good point - I forget them - thanks!

File: lib/EnsEMBL/REST/Model/ga4gh/ga4gh_utils.pm [hunk 1]
```diff
@@ -23,8 +23,9 @@ package EnsEMBL::REST::Model::ga4gh::ga4gh_utils;
 
 use Moose;
 extends 'Catalyst::Model';
-
+use Data::Dumper;
```
Comment (lairdm) [inline lib/EnsEMBL/REST/Model/ga4gh/ga4gh_utils.pm]: We probably want to pull Data::Dumper out before putting it in to production

**Comment by lairdm** [comment]
Other than the Data::Dumper, looks sane :)

File: lib/EnsEMBL/REST/Model/ga4gh/ga4gh_utils.pm (hunk 1 above)
Comment (sarahhunt) [inline lib/EnsEMBL/REST/Model/ga4gh/ga4gh_utils.pm]: Agreed!

**Pull request merged**